src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -328,11 +338,21 @@ public void testCreateCheckWithPartialPackageNameByBruteForce() throws Exception
     @SuppressWarnings("unchecked")
     public void testGenerateThirdPartyNameToFullModuleNameWithException() throws Exception {
         final URLClassLoader classLoader = mock(URLClassLoader.class);
-        when(classLoader.getURLs()).thenThrow(IOException.class);
-        final Method method = factory.getClass().getDeclaredMethod(
+        final Set<String> packages = new LinkedHashSet<>(1);
+        packages.add(BASE_PACKAGE);
```

Comment:
Please do it by stream https://stackoverflow.com/questions/2041778/how-to-initialize-hashset-values-by-construction, ones we jump to next java, it will be easy to update code, I think one inspection will hint this to update

Reply:
Done with `Collections.singleton`